crossing-training.lic

```diff
@@ -684,7 +689,8 @@ class CrossingTraining
     if DRSkill.getxp('Athletics') < 29
       walk_to(2245)
       if UserVars.athletics < 550
-        practice_branch
+        @climbing_target['target'] = 'branch'
```

Comment:
yea most likely it was a bad implementation. I've added the branch to the data and loaded that before going into the branch practice bit.